src/Broker/Broker.c

```diff
@@ -54,7 +54,7 @@ Broker_deliver_request(const enum RemoteInterface interface, const uint8_t* cons
 #elif defined BROKER_SEND_TELEMETRY
     const Packetizer_PacketType packet_type = Packetizer_PacketType_Telemetry;
 #else
-    const Packetizer_PacketType packet_type = Packetizer_PacketType_Telemetry;
+    const Packetizer_PacketType packet_type = Packetizer_PacketType_Telecommand;
```

Comment:
If nothing is set, both sending and receiving should expect the same packet type, otherwise, systems won't work "at default settings"

Reply:
done